packages/mobile/src/screens/upload-screen/fields/SubmenuList.tsx

```diff
@@ -1,28 +1,37 @@
 import { Children, cloneElement } from 'react'
 import type { ReactNode, ReactElement } from 'react'
 
+import type { StyleProp, ViewStyle } from 'react-native'
 import { View } from 'react-native'
 
 import { makeStyles } from 'app/styles'
 
 type SubmenuListProps = {
   children: ReactNode
   removeBottomDivider?: boolean
+  style?: StyleProp<ViewStyle>
 }
 
 const useStyles = makeStyles(({ spacing }) => ({
   root: {
     marginVertical: spacing(2)
+  },
+  content: {
+    marginHorizontal: spacing(2)
+  },
+  divider: {
+    marginHorizontal: spacing(-4)
```

Comment:
yeah i definitely agree here, i can consider what the options are here. since the parent is defining padding, it's either this just for the dividers, or every element inside the parent needs to specify the padding. what you think?

Reply:
My inclination would probably be to have something like `<FormSection>` and `<FormDivider>` where the section adds the padding and contains the other inputs. I do think it would be frustrating and possibly less maintainable to manage the padding separately for each input, even when compared to the negative margins on divider.

Reply:
I like that idea! Will incorporate